gometalinter.conf

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
:+1: is this type of configuration a new ability for gometalinter?

Reply:
Actually no idea! I just saw it today and assumed it has always been there